frontend/src/libs/constants/index.constant.ts

```diff
@@ -0,0 +1,4 @@
+const FIRST_INDEX = 0;
+const LAST_INDEX = -1;
+
+export { FIRST_INDEX, LAST_INDEX };
```

Comment:
We have such ones already in shared one, please check

Reply:
I have not found the file in "shared", I have visited some PRs with the "shared" tag, but I have not found them either, can you help me?